src/sentry/integrations/github/integration.py

```diff
@@ -155,43 +154,67 @@ def get_document_origin(org) -> str:
 # Github App docs and list of available endpoints
 # https://docs.github.com/en/rest/apps/installations
 # https://docs.github.com/en/rest/overview/endpoints-available-for-github-apps
-class GitHubIntegration(IntegrationInstallation, GitHubIssueBasic, RepositoryMixin, CommitContextIntegration):  # type: ignore[misc]
-    repo_search = True
+
+
+class GitHubIntegration(RepositoryIntegration, CommitContextIntegration, GitHubIssueBasic):  # type: ignore[misc]
     codeowners_locations = ["CODEOWNERS", ".github/CODEOWNERS", "docs/CODEOWNERS"]
```

Comment:
making this a property rather than leaving it here results in mypy issues

Reply:
Do other integrations have this too? I don't think mypy should be a blocker about making design decisions. Can you put up a PR with the issue and then maybe we can help fixing the typing?

Reply:
hmm i can tackle the mypy stuff after i get this in, if not i'll do what Gabe suggested